<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgOneBreakfasat</class>
 <widget class="QDialog" name="DlgOneBreakfasat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="spacing">
       <number>24</number>
      </property>
      <item row="9" column="0">
       <widget class="QLabel" name="lbGuest">
        <property name="text">
         <string>GUEST</string>
        </property>
       </widget>
      </item>
      <item row="4" column="6" colspan="2">
       <widget class="QPushButton" name="btnPayment">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/wallet-with-money.png</normaloff>:/images/wallet-with-money.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>PAYMENT MODE</string>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lbRoom">
        <property name="text">
         <string>ROOM</string>
        </property>
       </widget>
      </item>
      <item row="8" column="6" colspan="2">
       <widget class="QPushButton" name="btnRoom">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/dinner.png</normaloff>:/images/dinner.png</iconset>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>TOTAL</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>PAX</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lbCardType">
        <property name="text">
         <string>CARD TYPE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="8">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BREAKFAST</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="EQLineEdit" name="lePayment"/>
      </item>
      <item row="1" column="1">
       <widget class="EQLineEdit" name="lePax"/>
      </item>
      <item row="8" column="5">
       <widget class="EQLineEdit" name="leInvoice"/>
      </item>
      <item row="10" column="4">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" colspan="6">
       <widget class="EQLineEdit" name="leTotal"/>
      </item>
      <item row="11" column="0" colspan="8">
       <widget class="QWidget" name="widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>75</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QToolButton" name="btnOpenDraft">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Open
Draft</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnPrePrint">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Print
And Draft</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnPrint">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Print
And Save</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnPrintTax">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Print Tax</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/tax.png</normaloff>:/images/tax.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnExit">
           <property name="minimumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="text">
            <string>Exit</string>
           </property>
           <property name="icon">
            <iconset resource="res.qrc">
             <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>74</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnPax">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/guests.png</normaloff>:/images/guests.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <widget class="QPushButton" name="btnPrice">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="res.qrc">
          <normaloff>:/images/cash.png</normaloff>:/images/cash.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="EQLineEdit" name="lePrice">
        <property name="text">
         <string>6500</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>PRICE</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="5">
       <widget class="QLineEdit" name="leGuest"/>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="EQLineEdit" name="leRoom"/>
      </item>
      <item row="8" column="3" colspan="2">
       <widget class="QLabel" name="lbInvoice">
        <property name="text">
         <string>INVOICE</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lbCityLedger">
        <property name="text">
         <string>CITY LEDGER</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="7">
       <widget class="EQLineEdit" name="leCityLedger"/>
      </item>
      <item row="5" column="1" colspan="6">
       <widget class="EQLineEdit" name="leCardType"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
